lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.pop(-2)
 
 # Taking the result from the question above, what would `arr` now equal:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = '46'
 
```

Comment:
So if we take the array from the result above we have
arr = [12,34]
arr[arr.length + 2] really is saying arr[4] = 99
So if we set arr[4] to 99 , arr[2] and arr[3] are not defined, thus becoming nil
we end up with [12, 34, nil, nil, 99] !!!